### ISSUE TITLE
Solo.getViews() may return List with nulls in it

### ISSUE THREAD

**Issue by abutygin** [RobotiumTech/robotium#783]
It may occur on app screens with RecyclerView. 
In ViewFetcher.addChildren method viewGroup.getChildAt(i) can return null. So  lines:
```
else if(!onlySufficientlyVisible)
   views.add(child);
```
should be replaced by 
```
else if(!onlySufficientlyVisible && child != null)
   views.add(child);
```
or check child after viewGroup.getChildAt(i) :
```
final View child = viewGroup.getChildAt(i);
if (child == null) 
    continue;
```

**Comment by renas** [comment]
Thanks for reporting this. We'll make a new release shortly and will include this fix in it.

**Comment by renas** [comment]
Fixed in 5.2.2.

**Comment by abutygin** [comment]
Thanks a lot!
